_observing-your-data/metricsanalytics.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: default
+title: Metrics analytics
+nav_order: 40
+redirect_from:
+  - /observing-your-data/metricsanalytics/
+---
+
+# Metrics analytics
+Introduced 2.4
+{: .label .label-purple }
+
+With the release of OpenSearch 2.4, you can now ingest and visualize metrics data stored directly within OpenSearch using the **Metrics analytics** tool. This equips you with tools to analyze and correlate data across logs, traces, and metrics.
+
+Before this feature, you could only ingest and visualize logs and traces from your monitored environments. With **Metrics analytics**, you can now observe your digital assets with more granularity, gain deeper insight into the health of your infrastructure, and better inform your root cause analysis.
```

Comment:
```suggestion
Before the introduction of this feature, you could only ingest and visualize logs and traces from your monitored environments. With **Metric analytics**, you can now observe your digital assets with more granularity, gain deeper insight into the health of your infrastructure, and better inform your root cause analysis.
```

Reply:
The tool proper name is **Metrics**. I'll update bold name and use metric  in generic form.